search.py

```diff
@@ -0,0 +1,23 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+
+import sys
+import louis.db.api as api
+import louis.db as db
+from microbench import MicroBench
+import pandas as pd
+
+basic_bench = MicroBench()
+
+# Execute the SQL search function 
+@basic_bench
+def search(cursor, query):
+    return api.search_from_text_query(cursor, query)
+
+if __name__ == '__main__':
+    connection = db.connect_db()
+    with db.cursor(connection) as cursor:
+        results = search(cursor, ' '.join(sys.argv[1:]))
+        print(results)
```

Comment:
I would un-indent the print() here so it's outside the with block (we want to auto-close the cursor as soon as possible.

Reply:
I agree, done